cattle/plugins/docker/storage.py

```diff
@@ -129,7 +129,8 @@ def _do_image_activate(self, image, storage_pool, progress):
             temp = 'index.' + data.qualifiedName
         if progress is None:
             result = client.pull(repository=temp,
-                                 tag=data.tag, auth_config=auth_config)
+                                 tag=data.tag, auth_config=auth_config,
+                                 insecure_registry=True)
```

Comment:
Multiple grammatical errors in the comment and it's very wordy. Could this suffice?
```
# Always pass insecure_registry=True to prevent docker-py from pre-verifying the registry.
# Let the docker daemon handle the verification of and connection to the registry.
```